src/core_plugins/timelion/public/services/tick_formatters.js

```diff
@@ -0,0 +1,53 @@
+define(function (require) {
+
+  return function ticketFormatters() {
+    var formatters =  {
+      'bits': function (val, axis) {
+        var labels = ['b','kb','mb','gb','tb','pb'];
+        var index = 0;
+        while (val > 1000 && index < labels.length) {
+          val /= 1000;
+          index++;
+        }
+        return (Math.round(val * 100) / 100) + labels[index];
+      },
+      'bits/s': function (val, axis) {
+        var labels = ['b/s','kb/s','mb/s','gb/s','tb/s','pb/s'];
+        var index = 0;
+        while (val > 1000 && index < labels.length) {
```

Comment:
`val > 1000` should probably be `val >= 1000` otherwise you end up with stuff like this:

![screen shot 2016-11-14 at 12 45 52 pm](https://cloud.githubusercontent.com/assets/1250387/20280925/b4cfc2ba-aa6b-11e6-8323-275fc5c6a6bd.png)